src/base/bittorrent/torrenthandle.h

```diff
@@ -87,6 +87,14 @@ namespace BitTorrent
     class TrackerEntry;
     struct AddTorrentParams;
 
+    enum class FileAutoPriority
+    {
+        NAME_ASC,
```

Comment:
UPPER_CASED names is for constant only.
The old-style **enum** is in fact a declaration of several constants, so there are usually used UPPER_CASED names. IMO, for **enum class** we need to use CamelCased names. @sledgehammer999, what about it?
```c++
enum class FileAutoPriority
{
    NameAsc,
    NameDesc,
    SizeAsc,
    SizeDesc
};
```